modules/tinymce/src/themes/silver/main/ts/ui/throbber/Throbber.ts

```diff
@@ -103,12 +106,19 @@ const renderThrobber = (spec): AlloySpec => ({
   components: [ ]
 });
 
+const isPasteBinTarget = (target: unknown) =>
+  Optional.from(target)
+    .map(SugarElement.fromDom)
```

Comment:
This isn't going to work nicely when the target is an editor. Basically, we can't be converting to a SugarElement, without first knowing we're dealing with a `Node`.

Reply:
To clarify more, the type for `target` here should be `Editor | Node`

Reply:
Thanks for pointing this out. This has been reworked now with your provided snippets